photon-lib/src/test/scala/com/linkedin/photon/ml/hyperparameter/search/GaussianProcessSearchTest.scala

```diff
@@ -92,32 +94,19 @@ class GaussianProcessSearchTest {
 
     assertEquals(candidates1.length, n)
     assertEquals(candidates1.toSet.size, n)
-    assertTrue(candidates1.forall(_.params.toArray.forall(x => x >= lower && x <= upper)))
+    assertTrue(candidates1.forall(_.params.toArray.forall(x => x >= 0 && x < 1)))
   }
 
   @Test(dataProvider = "priorDataProvider", dependsOnMethods = Array[String]("testFindWithPriors"))
-  def testFindWithObservations(
-    currentCandidates: Seq[(DenseVector[Double], Double)],
-    priorCandidates: Seq[(DenseVector[Double], Double)],
-    testSetIndex: Int): Unit = {
-
-    val candidates1 = searcher.findWithObservations(n, currentCandidates)
-
-    assertEquals(candidates1.length, n)
-    assertTrue(candidates1.forall(_.params.toArray.forall(x => x >= lower && x <= upper)))
-    assertEquals(candidates1.size, n)
-  }
-
-  @Test(dataProvider = "priorDataProvider", dependsOnMethods = Array[String]("testFindWithObservations"))
   def testFindWithPastObservations(
     currentCandidates: Seq[(DenseVector[Double], Double)],
     priorCandidates: Seq[(DenseVector[Double], Double)],
-    testSetIndex: Int): Unit = {
+    estSetIndex: Int): Unit = {
```

Comment:
typo, t has been deleted. If the parameter is not used, you can delete it.

Reply:
Will delete it, thanks.